_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -26,7 +26,8 @@ This rule applies to any HTML `input`, `select` and `textarea` element with an `
 - **hidden**: the element is not [visible][], and not [included in the accessibility tree][]; or
 - **disabled**: the element is a [disabled element]; or
 - **fixed value**: the element is an `input` element with a `type` [attribute value][] of either `hidden`, `button`, `submit` or `reset`; or
-- **static**: the element is not part of [sequential focus navigation][] and has a [semantic role][] that is not a [widget role][].
+- **static**: the element is not part of [sequential focus navigation][] and has a [semantic role][] that is not a [widget role][]
+- the `autocomplete` attribute wears the [autofill expectation mantle](https://html.spec.whatwg.org/#autofill-expectation-mantle).
```

Comment:
I'm wondering about that. As per the specs that are linked:
> On an input element whose type attribute is in the Hidden state, the autocomplete attribute wears the autofill anchor mantle. In all other cases, it wears the autofill expectation mantle.

So this just means that the `input` element is not in the Hidden state (i.e. that the type attribute value is not `hidden` in our words). Then this is already handled by the **fixed value** condition above.

Reply:
Reading specs and the rest of the changes, I think we should rather explicitly exclude `autocomplete` which is either `on` or `off` (and nothing more).